x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.ts

```diff
@@ -148,28 +163,31 @@ export class ReportingAPIClient {
   /*
    * Diagnostic-related API calls
    */
-  public verifyConfig = (): Promise<DiagnoseResponse> =>
-    this.http.post(`${API_BASE_URL}/diagnose/config`, {
+  public async verifyConfig() {
+    return await this.http.post(`${API_BASE_URL}/diagnose/config`, {
```

Comment:
Just FWIW: these functions don't do anything with the call result so we don't need `async` `await` sugar. I think the return type gaurds have us covered.

Reply:
It's kind of been preached to me to use this way instead of "desugared promises." It makes stack traces more readable, and there are some optimizations in V8 that went into making async and await native. https://v8.dev/blog/fast-async